NeuroID/NeuroIDClass/NeuroID.swift

```diff
@@ -53,8 +57,9 @@ public class NeuroID: NSObject {
         set {}
     }
 
-    static var sendCollectionWorkItem: DispatchWorkItem?
+    static var _isSessionSampled: Bool = true
```

Comment:
Thoughts on `isSessionFlowSampled`? Since our primary use case is on the sessionFlow